kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/rest/RestURI.java

```diff
@@ -87,7 +87,7 @@
     // uris
     // process related prefixed by PROCESS_URI
     public static final String START_PROCESS_POST_URI = "{" + PROCESS_ID + "}/instances";
-    public static final String START_SYNC_PROCESS_POST_URI = "{" + PROCESS_ID + "}/syncInstances";
+    public static final String COMPUTE_PROCESS_OUTCOME_POST_URI = "{" + PROCESS_ID + "}/syncInstances";
```

Comment:
Shouldn't it be more accurate to use "computedInstances" in the path instead of syncInstances, with the new terminology?

Reply:
done